hotel_res.sql

```diff
@@ -59,10 +109,46 @@ CREATE TABLE employee
 	 foreign key (location_address) references location,
 	 foreign key (manager_id) references employee);
 
+	CREATE SEQUENCE e_seq;
```

Comment:
the Sequence keeps track of sequential numbers; it increments every time you call nextval. The trigger below goes off whenever there is an insert into the table it is set up for, and if that insert has a null value in id it inserts the next value in the sequence. This way we can do null inserts into employee, payment, and customer without having to randomly generate an id in the code (which has no guarantee of uniqueness, which will result in a failure on insert). Every insert just picks the next number in line